src/Sfx-Standalone/.build/tasks/publish.windows.js

```diff
@@ -56,23 +56,26 @@ gulp.task("publish:msi@build-msi",
         const packDirPath = path.resolve(path.join(buildInfos.paths.buildDir, packDirName));
         const publishDir = path.join(buildInfos.paths.publishDir, Platform.Windows);
         const filesWixPath = path.resolve(path.join(buildInfos.paths.buildDir, "files.msi.wxs"));
+        const filesWixXsltPath = path.resolve("./.build/files.msi.wxs.xslt");
         const wxsobjDir = path.resolve(path.join(buildInfos.paths.buildDir, "wxsobj"));
         const heatPath = path.resolve("./.vendor/wix/heat.exe");
         const candlePath = path.resolve("./.vendor/wix/candle.exe");
         const lightPath = path.resolve("./.vendor/wix/light.exe");
-        const heatCmd = utils.format("\"{}\" dir \"{}\" -ag -srd -cg MainComponentsGroup -dr INSTALLFOLDER -o \"{}\"", heatPath, packDirPath, filesWixPath);
+        const heatCmd = utils.format("\"{}\" dir \"{}\" -ag -srd -cg MainComponentsGroup -dr INSTALLFOLDER -o \"{}\" -t \"{}\"", heatPath, packDirPath, filesWixPath, filesWixXsltPath);
+        const updateGuidCmd = utils.format("powershell \"{}\" -XmlPath \"{}\"", path.resolve("./.build/tasks/publish.windows.update-guid.ps1"), filesWixPath);
         const candleCmd = utils.format("\"{}\" -arch x86 -out \"{}\\\\\" \"{}\" \"{}\"", candlePath, wxsobjDir, path.resolve(path.join(buildInfos.paths.buildDir, "msi.wxs")), filesWixPath);
         const lightCmd =
             utils.format(
-                "\"{}\" -b \"{}\" -spdb -out \"{}\" \"{}\" \"{}\"",
+                "\"{}\" -b \"{}\" -spdb -out \"{}\" \"{}\" \"{}\" -sice:ICE60 -sice:ICE91",

```

Comment:
ICE60 means files won't be uninstalled after user uninstall the MSI, right? Is it possible to fix them by adding <removefolder> for each folder in xslt?

Reply:
No, ICE60 means the font file needs a language specification but as SFX is not multi-language tool. 
https://docs.microsoft.com/en-us/windows/desktop/msi/ice60

Reply:
Got it. thanks.